README.md

```diff
@@ -90,6 +90,27 @@ octo.push('./bin/Sample.Web.3.2.1.tar.gz', {
 });
 ```
 
+#### Create release
+```js
+var octo = require('@octopusdeploy/octopackjs');
+
+octo.createRelease('./bin/Sample.Web.3.2.1.tar.gz', {
+        host: 'http://octopus-server/', 
+        apikey: 'API-XXXXXXXXX',
+        projectid: 'project-xxx',
```

Comment:
It looks like this (and channelId) are left over from a previous iteration and you need project and channel now (which is great as it lines up with octo.exe) rather than the ids?